### PR TITLE
Added validator --minimal option to action

### PR DESCRIPTION
**Pull request by ml-evs**
As it says on the tin.

No tests as its a simple argument (like skip optional), unless we really want them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ This can be chosen using the input `validator version`.
 | `port` | Port for the OPTIMADE URL. | Optional | `5000` | `v1+`
 | `protocol` | Protocol for the OPTIMADE URL. | Optional | `http` | `v1+`
 | `skip optional` | Whether or not to skip tests for optional features. | Optional | `false` | `v2+`
+| `minimal` | Whether or not to reduce the validation to a minimal test set that only checks responses and not filters. | Optional | `false` | `v2.4.1+`
```
Comment (CasperWA) [inline README.md:70]: I'd just release a new minor version, honestly.
```suggestion
| `minimal` | Whether or not to reduce the validation to a minimal test set that only checks responses and not filters. | Optional | `false` | `v2.5+`
```

**Review by CasperWA — CHANGES_REQUESTED** [review]
I'd vote to up the minor version, so that's the change I've suggested.
Also because we're moving to `v1.0.1`.

Looks good otherwise.

As a note, I think we should try to test all options though, at least just one test to make sure it doesn't fail and passes the correct option to `optimade-validator`. You don't need to test the content.

**Review by CasperWA — DISMISSED** [review]
There. Now the tests are fine 😅 

You can leave me out of the authorship.

**Comment by ml-evs** [comment]
> There. Now the tests are fine sweat_smile
> 
> You can leave me out of the authorship.

Huh, thanks, I guess `./run_tests.sh` locally doesn't run everything from the CI!

**Comment by CasperWA** [comment]
> Huh, thanks, I guess `./run_tests.sh` locally doesn't run everything from the CI!

It should run all the BATS tests. Don't know what else you mean?

**Pull request merged**